sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_server_events_v1/query.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  * REPLACE (
+    (
+      SELECT AS STRUCT
+        jsonPayload.* REPLACE (
+          (
+            SELECT AS STRUCT
+              jsonPayload.fields.* EXCEPT (device_id, user_id),
+              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
+              TO_HEX(SHA256(jsonPayload.fields.device_id)) AS device_id
+          ) AS fields
+        )
+    ) AS jsonPayload
+  ),
+  SPLIT(jsonPayload.logger, "-")[OFFSET(1)] fxa_log,
+FROM
+  `moz-fx-fxa-nonprod.gke_fxa_stage_log.stdout`
```

Comment:
I was expecting this to end up in the existing `moz-fx-fxa-nonprod-375e` project.  It looks like I don't have access to that dataset in the (new?) `moz-fx-fxa-nonprod` project.  Will all DEs get access, or only on an as-needed basis?

Reply:
I think by default not all DEs get access to it, but I'm sure you could ask for it since you are involved in this specific domain.

The project is also changing for the production data and will not include the random suffix.